packages/x-components/src/x-modules/popular-searches/wiring.ts

```diff
@@ -39,6 +39,20 @@ const retrievePopularSuggestions = wireDispatchWithoutPayload('fetchAndSaveSugge
  */
 export const setSearchedQueriesInPopularSearches = wireCommit('setSearchedQueries');
 
+/**
+ * Sets the popular searches state `params`.
+ *
+ * @public
+ */
+export const setRelatedTagsExtraParams = wireCommit('setParams');
+
+/**
+ * Sets empty value to the popular searches state `popularSearches`.
+ *
+ * @public
+ */
+export const setSuggestions = wireCommit('setSuggestions', []);
```

Comment:
You don't need this `setSuggestions`, you can remove it and the request is sent again when some extra params change.